src/shared/utils/UploadMedia.ts

```diff
@@ -0,0 +1,23 @@
+import Axios from "axios";
+
+const UploadMedia = async (files: FileList) => {
+  const formData = new FormData();
+  formData.append("file", files[0]);
+  formData.append("upload_preset", "ds2dkkvk");
+
+  try {
+    const data = await Axios.post(
```

Comment:
I would do something like `const { data: { version, publicId, format } }`

Reply:
That's much better, thanks!